packages/services/src/view-service/auctions.ts

```diff
@@ -65,12 +65,18 @@ export const auctions: Impl['auctions'] = async function* (req, ctx) {
     let state: DutchAuctionState | undefined;
     if (queryLatestState) state = (await querier.auction.auctionStateById(id))?.state;
```

Comment:
can you add a bracket here? I find this is easier to understand:

```typescript
    if (queryLatestState) {
      const auction = await querier.auction.auctionStateById(id);
      state = auction?.state;
    }
```

Reply:
Good call. Fixed.